Sources/EdgeNetworkHandlers/EdgeHitProcessor.swift

```diff
@@ -61,10 +61,22 @@ class EdgeHitProcessor: HitProcessing {
         }
 
         let event = edgeEntity.event
-        guard readyForEvent(event) else {
-            Log.debug(label: EdgeConstants.LOG_TAG, "\(SELF_TAG) - Not ready for event, will retry hit with id '\(entity.uniqueIdentifier)'.")
-            completion(false)
-            return
+        var edgeConfig: [String: String]
+
+        // Check which workflow is used to obtain configuration
+        if edgeEntity.configuration.isEmpty {
```

Comment:
recommend reversing the order of this condition and handling the most common case first, where the entity has config, and the old workflow after (rare post upgrade)